### PR TITLE
Pages/profile

### PR DESCRIPTION
**Pull request by numkhang**
###  UI

- สิ่งที่ทำ...
- other profile
- adjust navbar
- ViewAll from user/subject

### FUNCTION

- สิ่งที่ทำ...
- follow/unfollow
-navbar search ( autocomplete )

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -99,6 +100,7 @@ const App: React.FC = () => {
         <AuthRoute exact path="/linkAccount" component={LinkAccount} />
         <LayoutRoute exact path="/profile" component={Profile} />
         <LayoutRoute path="/viewAll/:id" component={ViewAll} />
+        <LayoutRoute path="/profile/:id" component={OtherProfile} />
```
Comment (JAmoMES) [inline src/App.tsx:103]: ใช้ userIdน่าจะดีกว่า

File: src/pages/OtherProfile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        .then(doc => {
+          doc.forEach(lecture => {
+            setLecture([...Lecture, { lectureId: lecture.id, ...lecture.data() } as LectureDTO])
+          })
+        })
+    }
+  }, [id])
+
+  const title = 'สรุปของ ' + Info.userName
+  const viewAllurl = '/viewAll/' + Info.userName + 'lecture' + id
```
Comment (JAmoMES) [inline src/pages/OtherProfile/index.tsx:62]: React.useMemo

File: src/pages/OtherProfile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  z-index: 100;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+export const OtherProfile: React.FC = () => {
+  const [Info, setInfo] = useState({} as MyUser)
```
Comment (JAmoMES) [inline src/pages/OtherProfile/index.tsx:27]: info

File: src/pages/OtherProfile/index.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  const [Lecture, setLecture] = useState([] as LectureDTO[])
```
Comment (JAmoMES) [inline src/pages/OtherProfile/index.tsx:28]: lecture

File: src/pages/ViewAll/index.tsx [hunk 5]
```diff
@@ -13,6 +13,7 @@ export const ViewAll: React.FC = () => {
   const history = useHistory()
   const { id } = useParams<{ id: string }>()
   const [viewAllLecture, setViewAllLecture] = useState<LectureDTO[]>([] as LectureDTO[])
+  const [Title, setTitle] = useState('')
```
Comment (JAmoMES) [inline src/pages/ViewAll/index.tsx:16]: title

**Comment by JAmoMES** [comment]
screenshot
ทำไมมันกว้างจังอะ navbar

<details><summary>Browser metadata</summary>





```
Path:      /home
Browser:   Chrome 92.0.4515.159 on Mac OS 10.15.7
Viewport:  1440 x 789 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-34--kushare.netlify.app%2Fhome&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=1440x789&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-34--kushare.netlify.app/home) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=61362e99c2bfc300077ae9cb&commentID=613722be56506b3f71d77497&resolution=resolved)

**Comment by numkhang** [comment]
> screenshot
> ทำไมมันกว้างจังอะ navbar
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-34--kushare.netlify.app/home) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=61362e99c2bfc300077ae9cb&commentID=613722be56506b3f71d77497&resolution=resolved)



> screenshot
> ทำไมมันกว้างจังอะ navbar
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-34--kushare.netlify.app/home) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=61362e99c2bfc300077ae9cb&commentID=613722be56506b3f71d77497&resolution=resolved)

ชื่อวิชามันยาวแล้วแสดงให้หมดมันอ่านง่ายกว่า

**Review by JAmoMES — APPROVED** [review]

**Pull request merged**